packages/components/src/font-size-picker/index.js

```diff
@@ -20,7 +20,12 @@ function getSelectValueFromFontSize( fontSizes, value ) {
 	return 'normal';
 }
 
-function getSelectOptions( optionsArray ) {
+function getSelectOptions( optionsArray, disableCustomFontSizes ) {
+	if(disableCustomFontSizes){
+		return [
+			...optionsArray.map( ( option ) => ( { value: option.slug, label: option.name } ) )
+		];
+	}
 	return [
 		...optionsArray.map( ( option ) => ( { value: option.slug, label: option.name } ) ),
 		{ value: 'custom', label: __( 'Custom' ) },
```

Comment:
Can you guard this line with the check instead or conditionally push to the object for the custom option to avoid code duplication?

Reply:
Yes, i can try that.